src/theme/breakpoints.ts

```diff
@@ -0,0 +1,6 @@
+export enum BREAKPOINTS {
+  WIDE = 420,
+  MEDIUM = 400,
+  SMALL = 375,
+  EXTRA_SMALL = 350,
+}
```

Comment:
@zzmp is this naming convention bad? should I follow interface norms and do  L M S XS

Reply:
You should follow interface norms. (I think this is fine, but if interface already has defined norms you should use those for consistency.)

Reply:
Since the interface naming convention is used for screen width, I think it actually could make sense to use a different naming convention here for widget width